dora/orientations/models.py

```diff
@@ -176,6 +184,63 @@ def get_referent_full_name(self):
             return full_name.strip()
         return self.referent_email
 
+    # Permet de renvoyer au besoin les valeurs dora ou d·i
+    def get_service_name(self):
+        return (
+            self.service.name
+            if self.service
+            else self.di_service_name
+            if self.di_service_id
+            else self.original_service_name
+        )
+
+    def get_contact_email(self):
+        return (
+            self.service.contact_email
+            if self.service
+            else self.di_contact_email
+            if self.di_service_id
+            else ""
+        )
```

Comment:
nit : _get is so Java_, `@property` ?

Reply:
Je comprends l'intérêt professé des property en cas de refacto, pour pouvoir passer d'un attribut à une méthode de façon transparente. En pratique, je préfère savoir que j'applique une méthode, et le `get` m'indique très clairement "attention, il se passe plus de choses ici que ce dont ça à l'air".
Après, c'est plus de la préférence personnelle qu'autre chose, et c'est toi qui va maintenir le code, donc si le get te dérange, ça vire sans aucun souci ;-)
Ça me fait penser que les [get_]contact_* sont mal nommés, _service_contact serait mieux. Donc de toute façon je repasse dessus, je peux mettre des properties si tu veux.

Reply:
non, rien à imposer. 
Pour ma part, ça ne me dérange pas qu'il y ait plusieurs manières d'exprimer une fonctionnalité. 
Au final, sur ce genre de sujet, je suis assez partisan du "chacun fait comme il veut". 
Du moment que c'est sans effet de bord, seule la lisibilité prime.